eeps/eep-0066.md

```diff
@@ -0,0 +1,432 @@
+  on the [String Content][].  The [String Delimiters][]
+  and escape characters work as for regular strings,
+  triple-quoted strings, or quoted atoms in Erlang.
+
+  So «`~"abc\d"`» is equivalent to «`<<"abc\d"/utf8>>`», and
+  «`~'abc"d'`» is equivalent to «`<<"abc\"d"/utf8>>`».
+
+  «`~"» would work as «`~s"`» and «`~"""`» would work
+  as «`~S"""`» below, regarding escape characters.
+
+  A simple way to create strings as UTF-8 binaries is supposedly
+  the first and most desired missing string feature in Erlang.
+  This sigil does just that and has no other features.
+
+* «`s`»: [string in Elixir][4].
+
+  Creates an Erlang `unicode:unicode_binary()`, handling
+  escape characters in the string content.  How other features
+  like string interpolation would work is still an open question.
+  
+  Escape characters and other features are the same regardless
+  of which [String Delimiters][] that are used.
+
+* «`S`»: [string in Elixir][4], verbatim.
+
+  Creates an Erlang `unicode:unicode_binary()`, with verbatim
+  string content in that only the [end delimiter][] character
+  can be escaped with a «`\`» character.  How other features
+  like string interpolation would work is still an open question.
+
+  Escape characters and other features are the same regardless
+  of which [String Delimiters][] that are used, except that
+  between triple-quote delimiters according to [EEP 64][]
+  there is no end delimiter character to escape.
+
+* «`r`»: regular expression.
+
+  Creates a term `{RE::unicode:charlist(),Flags::[unicode:latin1_char()]}`
+  that is an uncompiled regular expression with compile flags,
+  suitable for functions in the `re` module.  The `RE` element is
+  the [String Content][], and the `Flags` element is the [Sigil Suffix][].
+
+  See the [Regular Expressions][] section about the reasoning
+  behind this proposed term type.
+
+  Within the [String Content][], character escape sequences are handled
+  according to the regular expression rules, and in addition to them
+  the [end delimiter][] character can be escaped with a «`\`» character.
+  Between triple-quote delimiters according to [EEP 64][]
+  there is no end delimiter character to escape.
+
+### String Delimiters
+
+Immediately following the [Sigil Type][] is the string start delimiter.
+A specific start delimiter character has a corresponding
+end delimiter character.
+
+These are the start delimiter characters and they have themselves
+as end delimiter: single quote «`'`», and double quote «`"`».
+
+Triple-quote delimiters are also allowed, that is; a sequence of
+3 or more double quote «`"`» characters as described in [EEP 64][].
+
+### String Content
+
+Between the start and end [String Delimiters][], all characters
+are string content.
+
+In a triple-quoted string all characters are verbatim, but stripping
+of indentation and leading and trailing newline is done as usual
+as described in [EEP 64][].
+
+In a string with single character [String Delimiters][],
```

Comment:
This is consistent with triple-quoted code and single-quoted code
To me, it is intuitive